<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgGeneralBase</class>
 <widget class="QWidget" name="DlgGeneralBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">General settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox1_history_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Call History</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="m_pKeepHistory">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Keep history forever</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Automatically remove calls after</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KIntSpinBox" name="m_pHistoryMax">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>days</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="toolButton_historyClear">
          <property name="text">
           <string>Clear History</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="kcfg_displayHistoryStatus">
        <property name="text">
         <string>Display incoming, outgoing and missed for history items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_highlightMissedOutgoingCalls">
        <property name="text">
         <string>Highlight missed outgoing calls</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_highlightMissedIncomingCalls">
        <property name="text">
         <string>Highlight missed incoming calls</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_configGeneral_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>504</width>
       <height>171</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pKeepHistory</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pHistoryMax</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pHistoryMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DlgGeneralBase</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pKeepHistory</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgGeneralBase</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changed()</slot>
 </slots>
</ui>
